examples/canvas.rs

```diff
@@ -46,10 +46,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         canvas.place_at(Point::zero()).draw(&mut display)?;
 
         // prepare for drawing
-        let drawing_size = Size {
-            width: 100,
-            height: 100,
-        };
+        let drawing_size = Size {width: 100,height: 100};
```

Comment:
You can use `cargo fmt` to format your code, this shouldn't have been changed as it was already formatted.